### DISCUSSION TITLE
Performances, creating and saving.

### DISCUSSION THREAD

**Discussion by anycolorsmith** [probonopd/MiniDexed#239] [Q&A] [answered]
Hi, guys! Now my setup works correct and i have new question. I edit performans and save. How i can create another performances? And how many? Or each new performance will be saved on top of the previous one?

**Comment by craigyjp** [comment, marked as answer]
Currently its one only, but its in the pipeline to have multi performances up to 128 I guess

**Comment by anycolorsmith** [comment]
And this one start when power is on? And editable as text page?

**Reply by probonopd** [reply]
Yes.
Of course you can have mulitple ones on the SD card, but only the one named `performance.ini` will currently be loaded.

**Comment by Banana71** [comment]
I'm happy to read that the Performance mode is in the pipeline :-).  Additionally a "MasterVolume=xx" in Performance.ini would be a dream.

**Reply by probonopd** [reply]
Pull Requests welcome.

**Reply by dlbogdan** [reply]
I'll have a look when I have the time and branch this with mastervolume support on two gpio buttons and/or rotary encoder. 
Having had a look at the code reveals that setting this through midi commands is already supported. 
Adding an initial setting though for this in .ini files should be trivial.
